Package.swift

```diff
@@ -11,6 +11,7 @@ let package = Package(
     ],
   dependencies: [
     .package(url: "https://github.com/pointfreeco/swift-prelude.git", .revision("b70de46")),
+    .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", .revision("c510e7d")),
```

Comment:
I had to add this to fix this error on circle:

```
❌  /Users/distiller/project/Sources/PointFreeTestSupport/TestCase.swift:2:8: missing required module 'WKSnapshotConfigurationShim'

import SnapshotTesting
       ^
```

I dont really understand. it didnt happen locally.